xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-preferences/xwiki-platform-notifications-preferences-api/src/test/java/org/xwiki/notifications/preferences/internal/email/DefaultNotificationEmailUserPreferenceManagerTest.java

```diff
@@ -91,14 +93,16 @@ public void getDiffType() throws Exception
                 "NotificationAdministration"), new DocumentReference("mainWiki",
                 getCodeSpace(), "NotificationEmailPreferenceClass"),
                 "diffType")).thenReturn("NOTHING");
-        assertEquals(NotificationEmailDiffType.NOTHING, mocker.getComponentUnderTest().getDiffType(user2Id));
+        assertEquals(NotificationEmailDiffType.NOTHING,
+            mocker.getComponentUnderTest().getDiffType(userReferenceResolver.resolve(user2Id)));
 
         // Value from the user's wiki config
         when(documentAccessBridge.getProperty(new DocumentReference("someWiki", getCodeSpace(),
                         "NotificationAdministration"), new DocumentReference("mainWiki",
                         getCodeSpace(), "NotificationEmailPreferenceClass"),
                 "diffType")).thenReturn("NOTHING");
 
-        assertEquals(NotificationEmailDiffType.NOTHING, mocker.getComponentUnderTest().getDiffType(user2Id));
+        assertEquals(NotificationEmailDiffType.NOTHING,
+            mocker.getComponentUnderTest().getDiffType(userReferenceResolver.resolve(user2Id)));
```

Comment:
Sounds like you haven't added tests for the new `getInterval`, I understand you're in a rush but have you at least ran the module with quality profile (`mvn clean install -Pquality`) to ensure the coverage is  still good?

Reply:
Sounds good now that you have refactored the test :+1: